mlflow/store/artifact/cloud_artifact_repo.py

```diff
@@ -274,6 +274,11 @@ def _download_file(self, remote_file_path, local_path):
             or not file_size
             or file_size < MLFLOW_MULTIPART_DOWNLOAD_MINIMUM_FILE_SIZE.get()
             or is_fuse_or_uc_volumes_uri(local_path)
+            # DatabricksSDKModelsArtifactRepository can only download file via databricks sdk
+            # rather than presigned uri used in _parallelized_download_from_cloud. So we use
+            # DatabricksSDKModelsArtifactRepository._get_read_credential_infos(...) == None
+            # to fallback to _download_from_cloud.
+            or self._get_read_credential_infos([remote_file_path])[0] is None
```

Comment:
can we check if `self` is an instance of `DatabricksSDKModelsArtifactRepository` instead?